docker/openshift/faf-template.yml

```diff
@@ -171,3 +171,15 @@ parameters:
   name: FAF_VOLUME_CAPACITY
   required: true
   value: '1Gi'
+
+- displayName: 'Redis broker'
+  description: 'Address of the Redis message broker machine, with redis scheme, port number and db number'
+  name: RDS_BROKER
```

Comment:
This should be added below line 107.
```
- name: RDSBROKER
  value: ${RDS_BROKER}
- name: RDSBACKEND
  value: ${RDS_BACKEND}
```

Reply:
Yeah, I was just going to ask what else is missing in this file. Added.